packages/ag-charts-community/src/module/coreModules.ts

```diff
@@ -74,7 +74,8 @@ export type ExtensibleDefaults<
     ChartOptions = AgChartOptions & { series?: { type: SeriesType } },
 > = Extensible<ChartOptions>;
 
-export interface SeriesModule<SeriesType extends RequiredSeriesType = RequiredSeriesType> extends BaseModule {
+export interface SeriesModule<SeriesType extends RequiredSeriesType = RequiredSeriesType, SeriesOptions = any>
```

Comment:
Updated

Reply:
Think I just about got this to work, but had to stick an `any` over [here](https://github.com/ag-grid/ag-charts/pull/432/files#diff-0065008268af620210c1f943b2a92e6ed142dd2529e2083978ba120e6e476353R10)

Reply:
Ok - yeah the need to use `any` for these cases is annoying, but I think that is fine on balance since the likelihood is that the module is declared incorrectly rather than the infrastructure code doing the wrong thing 🤞